xarray/coding/variables.py

```diff
@@ -78,6 +78,71 @@ def __repr__(self) -> str:
         )
 
 
+class NativeEndiannessArray(indexing.ExplicitlyIndexedNDArrayMixin):
```

Comment:
It's good to mention that things have just been moved around without modifications, if that is the case. If not, its good to point out the differences for the reviewer.

Reply:
Yes will do, I'll put a comment in the code.

Reply:
Sorry for the confusion. A GitHub comment is enough, it's just for the review.

Reply:
These were just moved from conventions.py with typing added.